solutions/fully-configurable/variables.tf

```diff
@@ -0,0 +1,495 @@
+  default     = 2
+}
+
+variable "default_worker_pool_operating_system" {
+  type        = string
+  description = "The operating system installed on the worker nodes. [Learn more](https://cloud.ibm.com/docs/openshift?topic=openshift-vpc-flavors)"
+  default     = "RHEL_9_64"
+}
+
+variable "default_worker_pool_labels" {
+  type        = map(string)
+  description = "A set of key-value labels assigned to the worker pool for identification."
+  default     = {}
+}
+
+variable "default_worker_pool_secondary_storage" {
+  type        = string
+  description = "The secondary storage attached to the worker nodes. Secondary storage is immutable and can't be changed after provisioning."
+  default     = null
+}
+
+variable "enable_autoscaling_for_default_pool" {
+  type        = bool
+  description = "Set `true` to enable automatic scaling of worker based on workload demand."
+  default     = false
+}
+
+variable "default_pool_minimum_number_of_nodes" {
+  type        = number
+  description = "The minimum number of worker nodes allowed in the pool, ensuring at least one worker is always running."
+  default     = 1
+}
+
+variable "default_pool_maximum_number_of_nodes" {
+  type        = number
+  description = "The maximum number of worker nodes allowed in the pool, preventing the pool from exceeding three workers."
+  default     = 3
+}
+
+variable "additional_security_group_ids" {
+  type        = list(string)
+  description = "A list of security group IDs that are attached to the worker nodes for additional network security controls."
+  default     = []
+}
+
+variable "additional_worker_pools" {
+  type = list(object({
+    vpc_subnets = optional(list(object({
+      id         = string
+      zone       = string
+      cidr_block = string
+    })), [])
+    pool_name         = string
+    machine_type      = string
+    workers_per_zone  = number
+    resource_group_id = optional(string)
+    operating_system  = string
+    labels            = optional(map(string))
+    minSize           = optional(number)
+    secondary_storage = optional(string)
+    maxSize           = optional(number)
+    enableAutoscaling = optional(bool)
+    boot_volume_encryption_kms_config = optional(object({
+      crk             = string
+      kms_instance_id = string
+      kms_account_id  = optional(string)
+    }))
+    additional_security_group_ids = optional(list(string))
+  }))
+  description = "List of additional worker pools. [Learn more](https://github.com/terraform-ibm-modules/terraform-ibm-base-ocp-vpc/blob/main/solutions/fully-configurable/DA_docs.md#options-with-worker-pools)"
+  default     = []
+}
+
+##############################################################
+# COS Related
+##############################################################
+
+variable "existing_cos_instance_crn" {
+  type        = string
+  description = "The COS id of an already existing COS instance to use for OpenShift internal registry storage."
```

Comment:
COS id -> Don't use `COS`, use `Object Storage` in the description.  Also you mention `id` - change that to `CRN`

Reply:
Done